Source/WebCore/svg/SVGAnimationElement.cpp

```diff
@@ -393,7 +393,7 @@ static inline double solveEpsilon(double duration) { return 1 / (200 * duration)
 
 const Vector<float>& SVGAnimationElement::keyTimes() const
 {
-    return calcMode() == CalcMode::Paced ? m_keyTimesForPaced : m_keyTimesFromAttribute;
+    return (calcMode() == CalcMode::Paced && animationMode() != AnimationMode::Path) ? m_keyTimesForPaced : m_keyTimesFromAttribute;
```

Comment:
Nit: There is no need for the parentheses. The operator precedence should work as expected:

1. The equality != and == 
2. The logical &&
3. The ternary conditional ? :